actions.yml

```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
     contents: read
     pull-requests: read
     security-events: write
+ridedott/merge-me-action:
+  permissions:
+    contents: write
+shogo82148/actions-check-permissions:
+# https://github.com/saucelabs/sauce-connect-action/issues/24
+saucelabs/sauce-connect-action:
+  permissions:
+    contents: write
+codecov/codecov-action:
+  permissions:
+    contents: write # not required for public repos
```

Comment:
Is this means that private repo requires `write` permission?

Reply:
Related: https://github.com/codecov/codecov-action/issues/300

Reply:
I removed contents permission.